commons/clever-cloud-overview/hooks.md

```diff
@@ -42,6 +42,18 @@ possible with your build tool, but sometimes it's not possible. Another reason
 to use hooks is to integrate Clever Cloud-specific tasks which don't belong in
 the project's build configuration.
 
+## Exceptions
+
+Some instance type do not have hooks support or only some of them.
```

Comment:
instance types* (do we actually use "instance types" in the documentation elsewhere?)

Reply:
Yeah I wasn't sure about this one either. We use it in https://github.com/CleverCloud/doc.clever-cloud.com/blob/master/commons/clever-cloud-overview/common-application-configuration.md but that's pretty much it. Do you have a better naming idea to describe this?

Reply:
I guess "application type(s)" would be better, at least.

Reply:
updated